less/external-homepage/hp.less

```diff
@@ -906,3 +906,14 @@ body > a.sr-only-focusable { // "Skip to main content" link
     color: white;
   }
 }
+
+/* Fix unwanted border on HP mobile view buttons SBTWO-9448 */
+
```

Comment:
Need a max-width media query around this so it only applies on mobile, since more-links-trigger is always visible and sometimes wants its line